portal/src/graphql/adminapi/AuditLogList.tsx

```diff
@@ -31,6 +32,8 @@ export interface AuditLogListProps {
   pageSize: number;
   totalCount?: number;
   onChangeOffset?: (offset: number) => void;
+  onColumnClick?: () => void;
```

Comment:
Actually multiple column can be clickable. Use a more specific name, say `onToggleSortDirection`.